Toolset/palettes/script editor/behaviors/revsecommoneditorbehavior.livecodescript

```diff
@@ -694,6 +694,8 @@ private function textFormatSelection pText
       put return & tTextLines[tIndex] after tResult
    end repeat
    
+   if line - 1 of pText = "" then put cr after tResult
```

Comment:
There's the `- 1` thing again ;-) I have asked internally to see if others think we should be pedantic about this... it might be just me!